src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-slot/index.js

```diff
@@ -105,11 +106,10 @@ Component.register('sw-cms-slot', {
         onCloseElementModal() {
             this.showElementSelection = false;
         },
-
-        onSelectElement(elementType) {
-            this.element.data = {};
-            this.element.config = {};
-            this.element.type = elementType;
+        onSelectElement(element) {
```

Comment:
can you add a testcase for this change in the `Resources/app/administration/test/module/sw-cms/component/sw-cms-slot.spec.js` test file?

Reply:
Sure, I can write a corresponding test.

@keulinho Just out of interest: Why am I the only one who has to write tests? In other pull requests these are not requested and approved.